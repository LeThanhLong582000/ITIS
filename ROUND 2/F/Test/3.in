100
597 948 292
600 469 526
719 156 289
439 642 621
764 906 983
777 557 178
435 573 267
213 132 295
764 298 716
380 365 224
603 757 616
403 271 682
615 451 555
578 795 177
433 510 443
353 82 22
325 60 893
696 673 12
438 474 614
250 97 591
800 701 888
581 251 613
793 179 695
437 419 311
963 547 760
909 882 494
990 117 803
904 997 131
496 662 654
32 860 355
793 662 121
894 599 240
250 941 478
748 936 677
795 930 854
854 41 805
37 587 619
507 205 526
503 767 716
849 515 354
836 73 285
343 762 514
332 844 761
760 579 704
576 859 701
865 561 930
954 662 617
557 362 586
935 508 637
818 291 657
550 254 233
976 217 837
747 401 370
902 16 175
988 944 314
434 776 994
866 814 650
531 854 302
585 607 877
936 876 506
363 397 884
777 644 449
232 794 340
461 345 750
864 546 94
645 696 937
647 186 58
417 492 460
712 700 134
240 373 204
609 370 827
187 555 895
90 190 889
716 769 382
888 903 18
785 573 339
631 123 235
278 921 679
182 58 189
699 962 880
531 747 257
70 165 346
945 247 366
665 295 429
491 802 333
609 956 966
603 73 890
918 851 769
779 646 7
60 906 64
922 724 151
515 259 568
277 863 511
995 434 16
70 947 409
796 189 67
369 522 740
82 554 566
420 450 161
733 562 185
