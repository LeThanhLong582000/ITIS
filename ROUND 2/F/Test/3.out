0
329
290
209
421
179
65
188
591
120
231
407
360
0
183
23
579
13
289
372
494
472
655
165
588
261
804
19
244
0
122
241
0
245
360
806
35
414
226
344
286
48
125
443
209
617
455
391
532
592
234
798
358
176
179
326
351
0
428
283
425
291
0
433
95
443
59
193
73
36
533
264
395
165
2
276
236
18
157
309
21
126
367
400
11
310
710
418
8
0
152
412
0
17
0
68
294
47
66
178
